cluster/images/conformance/Makefile

```diff
@@ -56,15 +59,19 @@ endif
 
 	cd ${TEMP_DIR} && sed -i.back "s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
 
-	docker build --pull -t ${REGISTRY}/conformance-${ARCH}:${VERSION} ${TEMP_DIR}
-	rm -rf "${TEMP_DIR}"
+	# Must pre-pull image because it is not availabe on gcr mirror
+	# https://github.com/moby/buildkit/issues/1271
+	# https://github.com/kubernetes/kubernetes/pull/92042
+	docker pull $(BASEIMAGE)
 
-push: build
-	docker push ${REGISTRY}/conformance-${ARCH}:${VERSION}
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --pull --platform linux/$(ARCH) $(OUTPUT) -t ${REGISTRY}/conformance-${ARCH}:${VERSION} ${TEMP_DIR}
 ifeq ($(ARCH),amd64)
 	docker rmi ${REGISTRY}/conformance:${VERSION} 2>/dev/null || true
-	docker tag ${REGISTRY}/conformance-${ARCH}:${VERSION} ${REGISTRY}/conformance:${VERSION}
-	docker push ${REGISTRY}/conformance:${VERSION}
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --pull --platform linux/$(ARCH) $(OUTPUT) -t ${REGISTRY}/conformance:${VERSION} ${TEMP_DIR}
```

Comment:
we need to ensure buildx is initialized before trying to build, a clean docker install on linux doesn't have a builder

Reply:
@BenTheElder won't it use docker driver by default?

Reply:
Resolving this as we are moving away from using buildkit here